WatsonDeveloperCloud/Dialog/Tests/DialogTests.swift

```diff
@@ -467,7 +470,43 @@ class DialogTests: XCTestCase {
         waitForExpectation()
     }
     
-    // TODO: Get all conversation session history.
+    /// Get conversation history.
+    func getConversationHistory() {
+        let description = "Retrieving conversation history."
+        let expectation = expectationWithDescription(description)
+
+        let sydneyOffset = abs(NSTimeZone(name: "Australia/Sydney")!.secondsFromGMT)
+        let localOffset = abs(NSTimeZone.localTimeZone().secondsFromGMT)
+        let serverOffset = sydneyOffset + localOffset
+        let dateFromOffset: NSTimeInterval = -120.0 + Double(serverOffset)
```

Comment:
Good idea, but I think it's fine in this case because there is no user-specific configuration. Plists are also probably unnecessary for test code.